lib/i18n_fixes.rb

```diff
@@ -67,28 +67,3 @@ def locale
     end
   end
 end
-
```

Comment:
> This information should be in the commit message

Oh absolutely, was making notes here as was too tired yesterday to make enough sense for a commit message :)

> What prompted the change?

Mostly remembering that it was Rails 4.1 specific...

> Failing specs?

...and also that - it tries to access cache variables which were removed as one of the big things in Rails 4.2 was an overhaul of object caching so our patch causes breakages. I'll think of a sensible way to phrase this in a minute. One more coffee should do it